infrastructure/ansible/roles/traffic-vault/README.md

```diff
@@ -51,7 +51,7 @@ Example Playbook
       riak_listener_protobuf_internal: "{{ ansible_default_ipv4.address }}:8087"
       riak_protobuf_backlog: 4096
       riak_listener_https_internal: "{{ ansible_default_ipv4.address }}:8088"
-      riak_tls_protocols_tlsv1: 'on'
```

Comment:
This can be removed entirely with a better default

Reply:
Removed in 82fe83157b.